### PR TITLE
testing/yakuake: new aport

### PR DESCRIPTION
**Pull request by PureTryOut**

### CODE REVIEW

File: testing/yakuake/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+build() {
+	cmake \
+		-DCMAKE_BUILD_TYPE=RelWithDebInfo \
+		-DCMAKE_INSTALL_PREFIX=/usr \
+		-DCMAKE_INSTALL_LIBDIR=lib
+	make
+}
+
+check() {
+	CTEST_OUTPUT_ON_FAILURE=TRUE ctest
```
Comment (maxice8) [inline testing/yakuake/APKBUILD]: xvfb-run isn't used here (is it used implicitly ?)

Comment (PureTryOut) [reply testing/yakuake/APKBUILD]: Sorry my mistake, checkdepends is not needed. Fixed it.

**Comment by maxice8** [comment]
image

Getting this when installing yakuke, missing a dependency ?

**Comment by PureTryOut** [comment]
I knew I forgot something lol. Thanks, added the dependency.

**Pull request closed without merging**